pyproject.toml

```diff
@@ -29,45 +34,71 @@ maintainers = [
   {name = "Dimitri Yatsenko", email = "dimitri@datajoint.com"},
   {name = "DataJoint Contributors", email = "support@datajoint.com"},
 ]
-description = "A relational data pipeline framework."
+# manually sync here: https://datajoint.com/docs/core/datajoint-python/latest/#welcome-to-datajoint-for-python
+description = "DataJoint for Python is a framework for scientific workflow management based on relational principles. DataJoint is built on the foundation of the relational data model and prescribes a consistent method for organizing, populating, computing, and querying data."
 readme = "README.md"
 license = {file = "LICENSE.txt"}
 keywords = [
     "database",
-    "data pipelines",
-    "scientific computing",
-    "automated research workflows",
+    "automated",
+    "automation",
+    "compute",
+    "data",
+    "pipeline",
+    "workflow",
+    "scientific",
+    "science",
+    "research",
+    "neuroscience",
+    "bioinformatics",
+    "bio-informatics",
+    "datajoint",
 ]
+# https://pypi.org/classifiers/
 classifiers = [
-  "Programming Language :: Python"
-]
-
-[project.optional-dependencies]
-test = [
-  "pre-commit",
-  "pytest",
-  "pytest-cov",
-  "black==24.2.0",
-  "flake8",
+  "Programming Language :: Python",
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Science/Research",
+  "Intended Audience :: Healthcare Industry",
+  "License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+  "Topic :: Scientific/Engineering",
+  "Topic :: Scientific/Engineering :: Bio-Informatics",
+  "Topic :: Scientific/Engineering :: Neuroscience", # Not standard, but just in case
 ]
 
 [project.urls]
 Homepage = "https://datajoint.com/docs"
 Documentation = "https://datajoint.com/docs"
 Repository = "https://github.com/datajoint/datajoint-python"
 "Bug Tracker" = "https://github.com/datajoint/datajoint-python/issues"
-Changelog = "https://github.com/datajoint/datajoint-python/blob/master/CHANGELOG.md"
+"Release Notes" = "https://github.com/datajoint/datajoint-python/releases"
```

Comment:
We are going to use git-diff as changelog like https://github.com/minio/minio-py/compare/7.2.14...7.2.15, and it will be at the end of a release note like https://github.com/minio/minio-py/releases, so putting release notes here instead